app/components-react/highlighter/ClipTrimmer.tsx

```diff
@@ -151,8 +151,10 @@ export default function ClipTrimmer(props: { clip: IClip }) {
   function stopDragging() {
     if (isDragging.current === 'start') {
       HighlighterService.actions.setStartTrim(props.clip.path, localStartTrim);
+      UsageStatisticsService.actions.recordAnalyticsEvent('Highlighter', { type: 'Trim' });
     } else if (isDragging.current === 'end') {
       HighlighterService.actions.setEndTrim(props.clip.path, localEndTrim);
+      UsageStatisticsService.actions.recordAnalyticsEvent('Highlighter', { type: 'Trim' });
```

Comment:
just to confrim-- this is recording the same analytic twice for 2 different actions?